packages/cli/src/build-ipfs-connection.util.ts

```diff
@@ -0,0 +1,66 @@
+import IPFS from "ipfs-core"
+
+import dagJose from 'dag-jose'
+// @ts-ignore
+import multiformats from 'multiformats/basics'
+// @ts-ignore
+import legacy from 'multiformats/legacy'
+import ipfsClient from "ipfs-http-client"
+import { IpfsApi } from "@ceramicnetwork/common"
+
+multiformats.multicodec.add(dagJose)
+const dagJoseFormat = legacy(multiformats, dagJose.name)
+
+const IPFS_DHT_SERVER_MODE = process.env.IPFS_DHT_SERVER_MODE === 'true'
+
+const BOOTSTRAP_LIST = [
+    '/dns4/ipfs-dev.3boxlabs.com/tcp/4012/wss/p2p/QmeRWpCaEpQLY1yJFFHf7pFbZZapXoj5E4pKvUtyExMUKk',
+    '/dns4/ipfs-dev.ceramic.network/tcp/4012/wss/p2p/QmadvKqYyLMHYjnwswd2bHM3UCFp4F4oSFdobYUKSPjBaZ',
+    '/dns4/ipfs-dev-cas.3boxlabs.com/tcp/4012/wss/p2p/QmPHLQoWhK4CMPPgxGQxjNYEp1fMB8NPpoLaaR2VDMNbcr',
```

Comment:
Not sure we want to connect to these. Are they also configured with Clay @valmack ?

Reply:
These three nodes we should connect to only if we are on `dev-unstable`.

Reply:
aha, clear